stores/toml/store.go

```diff
@@ -0,0 +1,132 @@
+package toml //import "go.mozilla.org/sops/stores/toml"
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/BurntSushi/toml"
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/stores"
+)
+
+// Store handles storage of TOML data.
+type Store struct {
+}
+
+// LoadEncryptedFile loads an encrypted TOML secrets file onto a sops.Tree object
+func (store *Store) LoadEncryptedFile(in []byte) (sops.Tree, error) {
+	// Load metadata.
+	metadataHolder := stores.SopsFile{}
+	err := toml.Unmarshal(in, &metadataHolder)
+	if err != nil {
+		return sops.Tree{}, fmt.Errorf("Error unmarshalling metadata: %s", err)
```

Comment:
As a general rule, error messages should start with a lower case letter. Can you please fix this for all your errors?

Reply:
I was following style of the existing codebase:
<img width="1111" alt="Screen Shot 2019-09-16 at 10 32 58 PM" src="https://user-images.githubusercontent.com/139342/64992168-a3d73580-d8d3-11e9-94d9-a57882ac381e.png">

Perhaps we could revisit all the errors at once in a separate PR and utilize the new Go 1.13's error wrapping?